db/dbConnection.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+	/**
+	 * This function creates a connection to the database
+	 *
+	 * @return PDO
```

Comment:
I think on the return, you need to say what the return is - it pretty much just repeats the description but targets the output